examples/binding_affinity_prediction/utils.py

```diff
@@ -46,20 +64,36 @@ def load_dataset(args):
     """
     assert args['dataset'] in ['PDBBind'], 'Unexpected dataset {}'.format(args['dataset'])
     if args['dataset'] == 'PDBBind':
-        dataset = PDBBind(subset=args['subset'],
+        if args['model'] == 'PotentialNet': 
+            from functools import partial
+            from dgllife.utils import potentialNet_graph_construction_featurization
```

Comment:
"potentialNet" -> "PN" for consistency?

Reply:
I think we used "ACNN_graph_construction_and_featurization" previously while you now use "PN_graph_construction_featurization". Either you can change "ACNN_graph_construction_and_featurization" to "ACNN_graph_construction_featurization" or you can change "PN_graph_construction_featurization" to "PN_graph_construction_and_featurization".

Reply:
fixed